docs/tutorial-web-fork.md

```diff
@@ -0,0 +1,136 @@
+
+``custom/config.yaml`` must follow the structure of default configuration file `config-default.yaml`, and can specify all the same fields. At build time, the two configs are merged with fields in custom config overriding those in the default config. While it is recommended to use YAML format for the custom config file, JSON format is also supported (custom/config.json).
+
+A common change one may want to implement is to set a custom “[tcr](https://github.com/gnosis/dex-react/wiki/Config#tcr)field” for the app to connect to a custom TCR contract to keep a curated list of tokens visible on the interface.
+
+More information on setting up a custom config can be found on [wiki/Config](https://github.com/gnosis/dex-react/wiki/Config).
+
+Now, let’s try changing the project name and appId through a custom config file. Create a custom/config.yaml file containing:
+
+```
+# App Id
+#    Get yours at: https://github.com/gnosis/dex-react/wiki/App-Ids-for-Forks
+appId: <your-app-id>
+
+# Name of the app
+name: My Gnosis Protocol Web Fork
+```
+
+
+Then, restart the application with `yarn start`, and open your browser at [http://localhost:8080/about](http://localhost:8080/about).
+
+You should see the changes already in effect.
+
+![](assets/fork_config_change.png)
+
+
+## Making changes to the codebase directly or inside the `custom` directory
+
+The project implements an _alternative source directory_ to allow for easier override without changing the already present code. You should make changes mainly in this directory in order to easily incorporate any desired updates from the original project repository.
+
+All absolute import paths will try to be resolved first from `custom/<path>`, failing that from `src/<path>`, and finally from `node_modules/<path>`.
+
+Let’s illustrate this with an example. Given two files:
+
+*   `src/pages/About.tsx`
+*   `custom/pages/About.tsx`
+
+```
+import About from 'pages/About'
+// will try resolving from custom/ first and failing that from src/
+```
+
+This allows for easier plug-n-play customization and simplifies future merges from upstream. Let’s try making a custom about page:
+
+1. Copy `src/pages/About.tsx` to `custom/pages/About.tsx`
+```
+mkdir -p custom/pages
+cp src/pages/About.tsx custom/pages/About.tsx
+```
+2. Now, edit the newly created `custom/pages/About.tsx` like so:
+
+```
+   <AboutWrapper>
+   + <h1>Here will be a new About page</h1>
+     <h1>About {CONFIG.name}</h1>
+```
+
+3. As before, restart the application with `yarn start` and open your browser at [http://localhost:8080/about](http://localhost:8080/about). You should see the text you just added visible on the interface:
+
+
+![](assets/fork_add_text.png)
+
+
+Keep in mind that the _alternative source directory_ only works for absolute paths. If you want to change something in a file that is imported as `./&lt;path>/&lt;file>`, you will have to change it in place.
```

Comment:
Why replace `<` with `&lt;` but not `>` with `&gt;`
Also, I doubt you need any html entities here at all, as it's inside a code block or whatever markdown translates to
This likely is displayed as is on the page, `./&lt;path>/&lt;file>`
```suggestion
Keep in mind that the _alternative source directory_ only works for absolute paths. If you want to change something in a file that is imported as `./<path>/<file>`, you will have to change it in place.
```
Please check actual rendered result first, of course